conans/cli/commands/__init__.py

```diff
@@ -32,3 +32,44 @@ def default(self, o):
 def json_formatter(data):
     myjson = json.dumps(data, indent=4, cls=ConanJSONEncoder)
     return myjson
+
+
+def add_log_level_args(subparser):
+    # FIXME: Arguments name suggestions
+    subparser.add_argument("-oe", "--errors", action='store_true',
```

Comment:
I think I would go for one verbosity level argument, like ``-v=level``, otherwise this would be adding a lot of clutter to all command line help

Reply:
In the issue associated the people was super fun of the -v -vv etc so be prepared for the rejection, but I agree, I don't like so many args.

Reply:
Oh, wait, maybe a `-vv` is a `-v` argument with `v` value 🤔

Reply:
Yes, exactly, maybe you can alias the number of `v` after the first one to level values increments